librz/include/rz_util/rz_time.h

```diff
@@ -12,9 +12,14 @@
 
 #define ASCTIME_BUF_MINLEN 26
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 #ifdef _MSC_VER
 struct timeval;
 
+#undef timezone
```

Comment:
Was this necessary in your tests? Otherwise, LGTM!

Reply:
yes

Reply:
The reason I'm asking is that probably it is not too safe to undef a define defined by system header files.

Maybe we can rename it rz_timezone and use a 'define rz_timezone timezone' for the non-windows case. What do you think?

Reply:
i think ok